src/stream/src/executor/managed_state/join/mod.rs

```diff
@@ -204,53 +225,76 @@ pub struct JoinHashMap<K: HashKey, S: StateStore> {
     inner: JoinHashMapInner<K>,
     /// Data types of the join key columns
     join_key_data_types: Vec<DataType>,
-    /// Data types of all columns
-    col_data_types: Vec<DataType>,
     /// Null safe bitmap for each join pair
     null_matched: FixedBitSet,
-    /// Indices of the primary keys
-    pk_indices: Vec<usize>,
     /// Current epoch
     current_epoch: u64,
     /// State table
-    pub(crate) state_table: StateTable<S>,
+    state: TableInner<S>,
+    /// Degree table
```

Comment:
It seems that the concept of "degree" is not explained anywhere in the code base, which could be confusing to others at the first time to read these code. How about explain it here?

Reply:
Sure!